src/lib/device_tree/src/lib.rs

```diff
@@ -227,7 +227,7 @@ impl<'a> Iterator for FdtIterator<'a> {
                     let value = SectionReader::new(&self.dt.struct_block, self.cursor, len);
                     self.cursor = align4(self.cursor + len);
                     // Note: This performs a copy of the whole path_buf array!
-                    return Some(Entry::Property::<'a> { path: Path { depth: self.depth + 1, len: self.len_buf, buf: self.path_buf }, value: value });
+                    return Some(Entry::Property { path: Path { depth: self.depth + 1, len: self.len_buf, buf: self.path_buf }, value: value });
```

Comment:
Clippy suggested replacing `value: value` with simply `value` in the past, but I didn't want to make a PR just for that. Could you update that here?

Reply:
I created https://github.com/oreboot/oreboot/issues/282 to do it as separate PR (fix all existing lint errors and configure CI). Is it ok to not do this here?

Reply:
Thanks for filing #282! I like your more systematic plan. I think it's OK to do the `value` replacement here or for addressing #282.